app/src/main/kotlin/at/bitfire/davdroid/ui/AppSettingsScreen.kt

```diff
@@ -138,6 +153,10 @@ fun AppSettingsScreen(
     tasksAppName: String,
     tasksAppIcon: Drawable?,
     pushEndpoint: String?,
+    pushDistributors: List<String>?,
+    pushDistributorsData: Map<String, Pair<String, Drawable>>?,
```

Comment:
I'd prefer `pushDistributors` to be a `List<PushDistributorInfo>`, with `PushDistributorInfo` being a sub-class of the model that holds all required information (package name, icon, label).

So it's clean and we can at any time add or remove data.